backend/ts/src/main/scala/aqua/backend/ts/TypeScriptService.scala

```diff
@@ -111,6 +113,11 @@ case class TypeScriptService(srv: ServiceRes) {
       |        service = args[2];
       |    }
       |
+      |    let incorrectServiceDefinitions = missingFields(service, [${membersNames.map { n => s"'$n'" }.mkString(", ")}]);
```

Comment:
It's better to use `const` for every declaration which is not going to be reassigned